angler/config/system-proprietary-blobs-api24.txt

```diff
@@ -26,12 +31,17 @@ system/lib/libmmjpeg_interface.so
 system/lib/libqomx_core.so
 system/lib/libtinyxml.so
 system/lib/soundfx/libfmas.so
+system/lib64/hw/activity_recognition.angler.so
 system/lib64/hw/audio.primary.msm8994.so
+system/lib64/hw/context_hub.default.so
 system/lib64/hw/gps.msm8994.so
 system/lib64/hw/nfc_nci.angler.so
+system/lib64/hw/sensors.angler.so
+system/lib64/libhubconnection.so
 system/lib64/libManufacture.so
 system/lib64/libfilterpack_facedetect.so
 system/lib64/libgps.utils.so
 system/lib64/libloc_core.so
 system/lib64/libloc_eng.so
+system/lib64/libprotobuf-cpp-full.so
```

Comment:
How did you conclude that the `libprotobuf-cpp-full.so` from factory images is newer? I diff the factory image one with the AOSP generated one & they're identical:
```
$ md5sum out_release__7.0.0_r6/aosp_prod/target/product/angler/system/lib/libprotobuf-cpp-full.so ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/lib/libprotobuf-cpp-full.so 
f23007c7de8a2817f779493bde0c69db  out_release__7.0.0_r6/aosp_prod/target/product/angler/system/lib/libprotobuf-cpp-full.so
f23007c7de8a2817f779493bde0c69db  ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/lib/libprotobuf-cpp-full.so
```
I did some extra digging and the reason that the 64bit version is never build from AOSP despite being there is because no release module is depending on it and is marked as optional. The reason that Nexus images have the 64bit version is because there is another proprietary library, the `system/lib64/libfacenet.so`, which needs it. Since we don't include that library as part of the blobs list we don't need it.